server/apiCalls/transformers/memosCount.test.js

```diff
@@ -332,12 +333,12 @@ describe('Count memos and courses as one for one course with two course offering
     const offeringsWithMemos = [
       offering1WithDates({
         dates: datesAfterDeadlineBeforeStart,
-        memoValues: { ladokRoundIds: ['2', '3', '4'] },
+        memoValues: { applicationCodes: ['2', '3', '4'] },
         offeringValues: { offeringId: '1' },
```

Comment:
`offeringId`  has to be removed to be sure tests working properly

Reply:
Removed